### DISCUSSION TITLE
Question 5: PCA

### DISCUSSION THREAD

**Discussion by mybirth0407** [Boostcamp-AI-Tech-1-15/ddobokki_log#33] [1. AI Math]
PCA가 갖는 3가지 의의가 무엇이고, PCA가 어떻게 그런 역할을 하는지 말해주세용

**Comment by minji-o-j** [comment]
PCA는 전체 데이터들의 분포를 가장 잘 설명할 수 있는 방향벡터(주성분)를 찾아주는 역할을 합니다.

또한 이를 이용해서 차원축소를 한다면, 데이터 전체를 가장 잘 표현하는 형태로 근사할 수 있습니다. 따라서 이는 차원의 저주를 해결하는데에 사용할 수 있습니다.
(하나는 뭘까요..?)

분산이 크게 표현되는 고유벡터로 차원축소를 하게 되면, 
projection을 시켰을 때 데이터가 넓게 흩어지기 때문에 차원 축소시 data를 가장 잘 표현하게 됩니다.

**Reply by mybirth0407** [reply]
좋습니다! PCA는 분산이이 최대가 되도록 하는 축을 찾고, 그 축과 직교하고 분산이 최대가 되는 축을 찾아나가는 방식으로 데이터를 축소하는데요. 이 변환을 반대로 수행하면 데이터의 복원이 가능할 거구요 (축소->확장),  분산이 작은 축들을 noise로 가정하면, 이것들을 제거함으로서 일부 denoising 효과를 낼 수 있습니다! 그래서
1. 데이터 차원 축소
2. 데이터 복원
3. 노이즈 제거
라고 생각하시면 좋을 듯 해요!
